op-e2e/setup.go

```diff
@@ -844,6 +844,9 @@ func (cfg SystemConfig) Start(t *testing.T, _opts ...SystemConfigOption) (*Syste
 		Stopped:              sys.Cfg.DisableBatcher, // Batch submitter may be enabled later
 		BatchType:            batchType,
 		DataAvailabilityType: sys.Cfg.DataAvailabilityType,
+		// Add the compression stuff
+		//CompressionAlgo: derive.Zlib,
+		CompressionAlgo: derive.Brotli10,
```

Comment:
Ideally we add a new field `CompressionAlgo` to the config and use it here
```suggestion
		CompressionAlgo: sys.Cfg.CompressionAlgo,
```
Let's just set the default to `Zlib` for now, until we rolled Brotli10 out to mainnet with Fjord.

Reply:
OK let's add a Brotli op-e2e test in a follow-up https://github.com/ethereum-optimism/protocol-quest/issues/256